wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # set the template path
+    TEMPLATE_PATH = phishingpage.get(TEMPLATE_NAME)
+
+    # if template exists in database but not locally ask for download
+    if not phishingpage.exists(TEMPLATE_PATH):
+        # get user's response
+        response = raw_input("Template is available online. Do you wnat to "\
+        "downlaod it now? [Yy]")
```

Comment:
I'll get those fixed. By the way what do you think of the way I implemented it. Especially on line 885, I'd tried it with a try and an if statement but they seemed unnecessary. Love to hear your opinion on it.